openhie/package-starter-kit
# PLAT-524 docker login fix

NB!! Remember to build the `openhie/package-base` image before testing!

How to test: 
- make a small package in platform that uses a private image (I recommend `jembi/eth-openhim-backup-service`)
- build `openhie/package-base`, build `jembi/platform`
- ensure you have permission to download `jembi/eth-openhim-backup-service`
- ensure that you do `docker rmi jembi/eth-openhim-backup-service`
- build the cli binary and place in platform
- init the eth-openhim-backup-service

If the package launches, then the private image has downloaded, and the fix works!
